sky/provision/aws/utils.py

```diff
@@ -0,0 +1,20 @@
+"""Utils for AWS provisioner."""
+from typing import Any
+from botocore import config
+
+from sky.adaptors import aws
+
+BOTO_MAX_RETRIES = 12
+
+
+def create_resource(resource: str,
+                    region: str,
+                    max_attempts: int = BOTO_MAX_RETRIES,
+                    **credentials) -> Any:
+    """Create an AWS resource."""
+    # overhead: 6.69 ms ± 41.5 µs
+    return aws.resource(
+        resource,
+        region=region,
+        config=config.Config(retries={'max_attempts': max_attempts}),
+        **credentials)
```

Comment:
This function is a bit shallow. Should we just move this `max_attempts` argument into the `adaptors.aws.resource` function?